.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@
 
 - [ ] I've given this PR a concise, self-descriptive, and meaningful title
 - [ ] I've linked relevant issues in the PR body
-- [ ] I've applied [the relevant labels](https://www.neuro.polymtl.ca/software/contributing#pr_labels) to this PR
+- [ ] I've applied [the relevant labels](https://intranet.neuro.polymtl.ca/geek-tips/contributing#pr-labels-a-href-pr-labels-id-pr-labels-a) to this PR
```

Comment:
> Labeling protocols may vary between NeuroPoly projects

This link is really generic and I'm not sure it resolve the question of a newcomer that want to know what are the labels commonly use and what is the best way to label its PR to help core maintainer keep things rolling.

Don't we have a more specific list or more details for SCT common practice ?

Reply:
So, just to give some context, here. About 2 years ago, each NeuroPoly project (SCT, ivadomed, shimming-toolbox, axondeepseg, etc.) had its own guidelines. The guidelines were mostly the same, with little tweaks here and there. So, we tried to centralize all of those duplicate documents in one place (The "Contributing Guidelines").

However, in hindsight, I think that you are absolutely right. The "centralized guidelines" became very generic, to the point where they became less helpful.

So, I have actually been wanting to reverse our "centralization" and go back to having an SCT-specific document. So, I am happy that you've brought this up. :heart:

Reply:
I think for now, though, this PR is quick enough to approve and merge, since it's just a link change.

But as a more long-term solution, I think it would be helpful to bring the contributing guidelines back to `CONTRIBUTING.rst` and/or our [internal developer wiki](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki).

Reply:
I created issue #3648 to track the bigger task of moving the documentation over, with a note to update the links in the PR template once this is done. Meanwhile, I did find the existing generic documentation very useful, because it explained the colour scheme for the different labels, and how many of each to apply to a PR. Beyond that, it seems like the only thing missing is a link to [the list of labels for this repository](https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/labels), where each label has an attached description.